### ISSUE TITLE
Link error on windows

### ISSUE THREAD

**Issue by aclysma** [rust-skia/rust-skia#431]
I tried to build on windows and hit the linker error below. I tried several version and it seems to happen from 0.34.0 onward (0.33.0 builds for me). Any ideas?

```
error: linking with `link.exe` failed: exit code: 1120

  = note:    Creating library C:\dev\rust\skulpin\target\debug\examples\sdl2_renderer_only.lib and object C:\dev\rust\skulpin\target\debug\examples\sdl2_renderer_only.exp
          skia.lib(icu.SkLoadICU.obj) : error LNK2019: unresolved external symbol __imp___std_init_once_begin_initialize referenced in function "bool __cdecl SkLoadICU(void)" (?SkLoadICU@@YA_NXZ)
          skia.lib(icu.umutex.obj) : error LNK2001: unresolved external symbol __imp___std_init_once_begin_initialize
          skia.lib(icu.SkLoadICU.obj) : error LNK2019: unresolved external symbol __imp___std_init_once_complete referenced in function "bool __cdecl SkLoadICU(void)" (?SkLoadICU@@YA_NXZ)
          skia.lib(icu.umutex.obj) : error LNK2001: unresolved external symbol __imp___std_init_once_complete
          C:\dev\rust\skulpin\target\debug\examples\sdl2_renderer_only.exe : fatal error LNK1120: 2 unresolved externals
```

**Comment by pragmatrix** [comment]
I can't reproduce this with you the current master of skulpin and

`cargo run -vv --release --example sdl2_renderer_only`

which downloads the rust-skia binaries version 0.36.0, and runs fine.

I suspect that the problem comes from using an older version of the Visual Studio / Visual Studio Build Tools. Can you see if an update fixes the linker problems (I've tried with 16.8.0 and 16.8.2)?

Related: https://github.com/Kethku/neovide/issues/399

**Comment by aclysma** [comment]
Thanks, updating VS build tools fixed it!
